docs/content/_index.md

```diff
@@ -48,20 +48,20 @@ of the execution when triggered by a `Pull Request` or a `Push`.
 
 ## Getting Started
 
-The easiest way to get started is to use the `tkn pac` CLI and its [bootstrap](/docs/guide/cli/#commands) command.
+The easiest way to get started is to use the `tkn pac` CLI and its [bootstrap](./docs/guide/cli.md#install) command.
```

Comment:
those links are not going to work on hugo :-D you can see the preview here : 

https://preview-pipelines-as-code-ci.apps.paac.devcluster.openshift.com/docs/eaa14f68dd791478a036cd54675f70df04cbc096/docs/guide/cli.md#install

:-D 

the best way to develop docs and preview the changes is to do : 

```
cd docs
hugo server
```

go to https://localhost:1313 and your change should automatically be displayed in browser live....

Reply:
yeah they are working but only via hugo generated :)